web_server/entity/xml_parser.py

```diff
@@ -49,17 +63,35 @@ def generate_vtt(self, test_name):
         """
         vtt_list = ['WEBVTT\n\n\n']
         elapse_time = timedelta()
+        if self.obj is not None:
+            for suite1 in self.obj.robot.suite.suite:
+                try:
```

Comment:
This code is not ease to read, consider moving the try except block to an initialization method.

Reply:
Hello Luis, I tried moving the try except block to the initialization method, however, since there might be multiple suites in self.obj.robot.suite.suite, sometimes it gives attribute error

Reply:
Also, there are some suites that has zero test cases in them, to adjust that, I am afraid we still need try except block in functions like get_test_names to ignore those

Reply:
Are the test case failing?

Reply:
Sadly yes... For single_suite, it passses; but it cannot pass for multiple_suites